incubator/neo4j/templates/loadbalancer.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "fullname" . }}-loadbalancer"
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "name" . }}
+    component: "{{ .Values.loadBalancerComponent }}"
+spec:
+  type: {{ .Values.serviceType }}
+  ports:
+    - port: {{ .Values.httpPort }}
+      targetPort: {{ .Values.httpPort }}
+      name: browser
+    - port: {{ .Values.boltPort }}
+      targetPort: {{ .Values.boltPort }}
+      name: bolt
+  selector:
+    app: {{ template "name" . }}
```

Comment:
The load balancer was used for both of the other components so I wasn't sure how to label it. 

Thinking about it I don't really need it so I'll delete it.